mbed/mbed.py

```diff
@@ -34,6 +34,7 @@
 import zipfile
 import argparse
 import tempfile
+from mbed_cdc import mbed_cdc
```

Comment:
This import should probably be scoped in some way. maybe `from mbed.mbed_cdc` or `from .mbed_cdc`. Further, this seems very redundant: `from thing import thing`.

Reply:
This is not a must have, this works in python 2